pkg/cmd/cli/cmd/importer/dockercompose.go

```diff
@@ -26,7 +26,7 @@ import (
 
 const (
 	dockerComposeLong = `
-Import a Docker Compose file as OpenShift objects
+Import a Docker Compose file as OpenShift objects [EXPERIMENTAL]
```

Comment:
Might be better to put a sentence at the end of the description, more of a blurb and more likely a user will see it.  I think we should go through and have similar sentences for app.json, David's new patch command, and anything else we consider new.